src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -37,13 +48,15 @@ public class DisputesController : ControllerBase
     /// <param name="hashids"></param>
     /// <param name="tokenEncoder"></param>
     /// <exception cref="ArgumentNullException"> <paramref name="mediator"/> or <paramref name="logger"/> is null.</exception>
-    public DisputesController(IBus bus, IMediator mediator, ILogger<DisputesController> logger, IHashids hashids, IDisputeEmailVerificationTokenEncoder tokenEncoder)
+    public DisputesController(IBus bus, IMediator mediator, ILogger<DisputesController> logger, IHashids hashids, IDisputeEmailVerificationTokenEncoder tokenEncoder, IOAuthUserService oAuthUserService, IMapper mapper)

```

Comment:
Fix the build warnings. Add the document parameters

> Parameter 'oAuthUserService' has no matching param tag in the XML comment for 'DisputesController.DisputesController(IBus, IMediator, ILogger<DisputesController>, IHashids, IDisputeEmailVerificationTokenEncoder, IOAuthUserService, IMapper)' (but other parameters do)

Reply:
ok